chalice/app.py

```diff
@@ -121,13 +121,10 @@ def dispatch(*args, **kwargs):
         )
 
         def maybe_fake_504() -> typing.Optional[chalice.Response]:
-            cookies = SimpleCookie(app.current_request.headers.get("Cookie", ""))
```

Comment:
Please remove the SimpleCookie import now that it's unused.

Reply:
Done.